awswrangler/athena/_read.py

```diff
@@ -95,37 +97,33 @@ def _fetch_parquet_result(
 ) -> Union[pd.DataFrame, Iterator[pd.DataFrame]]:
     ret: Union[pd.DataFrame, Iterator[pd.DataFrame]]
     chunked: Union[bool, int] = False if chunksize is None else chunksize
-    _logger.debug("chunked: %s", chunked)
+    _logger.debug("Chunked: %s", chunked)
     if query_metadata.manifest_location is None:
         return _empty_dataframe_response(bool(chunked), query_metadata)
     manifest_path: str = query_metadata.manifest_location
     metadata_path: str = manifest_path.replace("-manifest.csv", ".metadata")
-    _logger.debug("manifest_path: %s", manifest_path)
-    _logger.debug("metadata_path: %s", metadata_path)
+    _logger.debug("Manifest path: %s", manifest_path)
+    _logger.debug("Metadata path: %s", metadata_path)
     paths: List[str] = _extract_ctas_manifest_paths(path=manifest_path, boto3_session=boto3_session)
+    _logger.debug("Read %d paths from the manifest", len(paths))
```

Comment:
Wouldn't this be redundant with the log call inside `_extract_ctas_manifest_paths`?

Reply:
It would, thanks, fixed